azurelinuxagent/ga/update.py

```diff
@@ -267,6 +268,24 @@ def run(self, debug=False): # pylint: disable=R0912,R0914
             logger.info(os_info_msg)
             add_event(AGENT_NAME, op=WALAEventOperation.OSInfo, message=os_info_msg)
 
+            #
```

Comment:
initialization of these thread was racing against initialization of the main thread